book/src/leader-rotation.md

```diff
@@ -3,17 +3,46 @@
 At any given moment, a cluster expects only one fullnode to produce ledger
 entries. By having only one leader at a time, all validators are able to replay
 identical copies of the ledger. The drawback of only one leader at a time,
-however, is that a malicious leader is cabable of censoring votes and
+however, is that a malicious leader is capable of censoring votes and
 transactions. Since censoring cannot be distinguished from the network dropping
 packets, the cluster cannot simply elect a single node to hold the leader role
-indefinitely. Instead, the cluster minimizes the influence of a malcioius
+indefinitely. Instead, the cluster minimizes the influence of a malicious
 leader by rotating which node takes the lead.
 
 Each validator selects the expected leader using the same algorithm, described
 below. When the validator receives a new signed ledger entry, it can be certain
 that entry was produced by the expected leader.
 
-## Leader Schedule Generation
+## Leader Schedule Rotation
+
+Leader schedule is generated every epoch when the root fork crosses the
+epoch boundary.
```

Comment:
Is this tenable?  Is the root fork may skip the boundary, right?

Reply:
@rob-solana read further on :).